ramls/schemas/profiles/transformations.json

```diff
@@ -27,8 +27,8 @@
       "type": "string",
       "$ref": "recordTypes.json"
     },
-    "hasSameTagInHoldings": {
-      "description": "The sign that means the current transformation intended for both item and holding",
+    "hasSameTagInItems": {
```

Comment:
This is still a breaking change, I don't know how this PR is different from the previous one, but we still need to discuss it. This will need a migration script

Reply:
@KVupp ideally, this can be not broken changes, as this field is only for backend usage, it calculates at the backend, and used there, we don`t need it on UI side, or in the DB. Just need to ensure that these fields are not present in the DB (on the rancher, maybe someone created a mapping profile by the postman, and add this field in transformation, that`s why we had those issue). But we need to ensure that this field does not get to the DB, if present there - then this is a breaking change in my opinion

Reply:
I will reach Rancher to see mapping profiles, I don't think this field can get the DB

Reply:
@KVupp  @andrii-novytskyi the script is in place